schema/sale_artwork.js

```diff
@@ -45,8 +45,9 @@ const SaleArtworkType = new GraphQLObjectType({
               key: sale.increment_strategy,
             }).then(incrs => {
               // We already have the asking price for the lot. Produce a list
-              // of increments beyond that amount.
-              const tiers = incrs[0].increments
+              // of increments beyond that amount. Make a local copy of the
+              // tiers to avoid mutating the cached value.
+              const tiers = incrs[0].increments.slice(0)
```

Comment:
If you want to copy the whole array, can't you just do `slice()` without any args? (not a blocker, just curious if I'm missing something)

Reply:
Yep, you're right. I forget my idioms!

With ES6 splats, do you think it would be clearer to do `const tiers = [...incrs[0].increments]`?